py/packages/corpora/views.py

```diff
@@ -7,15 +7,15 @@
 class BuildBinaryView(View):
     def get(self, request, *args, **kwargs):
         try:
-            root_dir = "/workspace"
-            cli_main = "/workspace/py/packages/corpora_cli/main.py"
-            output_path = "/workspace/dist/corpora"
+            # root_dir = "/workspace"
+            # cli_main = "/workspace/py/packages/corpora_cli/main.py"
+            output_path = "/workspace/dist/corpora_static"
 
-            subprocess.run(
-                ["pyinstaller", "--onefile", cli_main, "--name", "corpora"],
-                check=True,
-                cwd=root_dir,
-            )
+            # subprocess.run(
```

Comment:
Next we need to do a real static binary - maybe with musl and nuitka .. or ... ?